doc/sphinx/conf.py.in

```diff
@@ -178,7 +178,7 @@ man_pages = [
     ("docs/fpga_tools/mmlink/mmlink", 'mmlink', u'Enable remote SignalTAP debugging', [author], 8),
     ("docs/fpga_tools/hssi/hssi", 'hssi', u'High Speed Serial Interface', [author], 8),
     ("docs/fpga_tools/opaevfio/opaevfio", 'opaevfio', u'Bind/Unbind accelerator to/from vfio-pci', [author], 8),
-    ("docs/fpga_tools/opaeuio/opaeuio", 'opaeuio', u'Bind/Unbind DFL device to/from dfl-uio-pdev', [author], 8),
+    #("docs/fpga_tools/opaeuio/opaeuio", 'opaeuio', u'Bind/Unbind DFL device to/from dfl-uio-pdev', [author], 8),
```

Comment:
Why `#`? Git keeps the line for future use :)

Reply:
I was following precedent that had been established on lines 174-177 of the same file.